src/component/admin/auth/AutoCreateForm/AutoCreateForm.tsx

```diff
@@ -1,102 +1,120 @@
 import React, { ChangeEvent, Fragment } from 'react';
-import { FormControl, Grid, MenuItem, Switch, TextField, Select, InputLabel, FormControlLabel } from '@material-ui/core';
+import {
+    FormControl,
+    Grid,
+    MenuItem,
+    Switch,
+    TextField,
+    Select,
+    InputLabel,
+    FormControlLabel,
+} from '@material-ui/core';
 
-interface Props { 
+interface Props {
     data?: {
         enabled: boolean;
         autoCreate: boolean;
         defaultRootRole?: string;
         emailDomains?: string;
-
     };
     setValue: (name: string, value: string | boolean) => void;
 }
 
-function AutoCreateForm({ data = { enabled: false, autoCreate: false }, setValue }: Props) {
+function AutoCreateForm({
```

Comment:
Small detail, but perhaps we could change this to an arrow function and explicit export?